pkg/cluster-runtime/installer.go

```diff
@@ -226,3 +238,24 @@ func (i *Installer) GetCurrentDriver() (registry.Driver, runtime.Driver, error)
 
 	return registryDriver, runtimeDriver, nil
 }
+
+func (i *Installer) setNodeTaints(hosts []net.IP, driver runtime.Driver) error {
+	var node k8sv1.Node
+	nodeList := k8sv1.NodeList{}
+	if err := driver.List(context.TODO(), &nodeList); err != nil {
+		return fmt.Errorf("failed to list cluster nodes: %v", err)
+	}
+
+	for _, node = range nodeList.Items {
+		for _, ip := range hosts {
```

Comment:
bug: hosts is a subset for nodelist ,so we need to find the target host ip to do taint action